lib/oppen.rb

```diff
@@ -12,13 +12,16 @@ module Oppen
   # Entry point of the pretty printer.
   #
   # @param config [Config]
+  # @param genspace [String/Proc] space generator: can be a string, a lambda, a block, ...
   # @param margin [Integer] maximum line width desired
   # @param new_line [String] the delimiter between lines
+  # @param out [Buffer] should have a write method
```

Comment:
`Buffer` is `IO::Buffer`. This is not what we require. We just want it to be an `Object`. The explanation next to it is sufficient.